### PR TITLE
docs: Add Email Notifications info

### PR DESCRIPTION
**Pull request by aviv926**
This PR adds information to the documentation about the email notification options in Immich.

Would we like to include this information in the Administration section or under Features?

In #9736, if I understood correctly then we include user preferences for email messages, would we like to add information about this?

### CODE REVIEW

File: docs/docs/administration/email-notification.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+# Email Notifications
+
+Immich supports the option to send notifications via email, this option is supported for:
+
+- Creating a new user .
+- A user is added to a shared album (send email to the newly shared user).
+- A shared album is added with new assets (send an email to all shared users beside the one that is adding assets).
+
+The easiest way to set up a mail server for notifications is through Google, you may want to avoid using Gmail for privacy reasons or as part of your degoogle plan, you can use any SMTP server that can support login using name and password.
```
Comment (alextran1502) [inline docs/docs/administration/email-notification.md:12]: Maybe this will be better in the Guide section. In this section we just describe that is the Email notification feature, how can we set it up, what are the parameter...etc

File: docs/docs/administration/email-notification.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Create an app password
+
+First, it is recommended to create a dedicated Google account for your Immich instance, and not to mix it with your main Google account.
```
Comment (alextran1502) [inline docs/docs/administration/email-notification.md:16]: Is this your opinion? We should be subjective in writing documentation

File: docs/docs/administration/email-notification.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Immich does not support this setting through the mobile app at this time.
+
+</TabItem>
+</Tabs>
+
+---
+
+Now for each new user created in the system, the system administrator will be able to choose whether to send an email informing the user of the new account.
+
+:::danger plain password
```
Comment (alextran1502) [inline docs/docs/administration/email-notification.md:63]: This information is not related to the email notification feature. It can be included in the user creation session. Try to keep the documentation short and to the point

File: docs/docs/administration/email-notification.md (hunk 2 above)
Comment (danieldietzler) [reply docs/docs/administration/email-notification.md:16]: We should *not* be subjective :D

File: docs/docs/administration/email-notification.md (hunk 3 above)
Comment (aviv926) [reply docs/docs/administration/email-notification.md:63]: Agreed, I fixed it.

File: docs/docs/administration/email-notification.md (hunk 2 above)
Comment (aviv926) [reply docs/docs/administration/email-notification.md:16]: I put it in a :::tip:::, if you think it should be removed, tell me.

File: docs/docs/administration/email-notification.md (hunk 1 above)
Comment (aviv926) [reply docs/docs/administration/email-notification.md:12]: Maybe remove this section?

File: docs/docs/administration/email-notification.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Email Notifications
+
+Immich supports the option to send notifications via email, this option is supported for:
```
Comment (danieldietzler) [inline docs/docs/administration/email-notification.md:6]:
```suggestion
Immich supports the option to send notifications via email. Currently, this is supported for:
```

File: docs/docs/administration/email-notification.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+- Creating a new user .
+- A user is added to a shared album (send email to the newly shared user).
+- A shared album is added with new assets (send an email to all shared users beside the one that is adding assets).
```
Comment (danieldietzler) [inline docs/docs/administration/email-notification.md:10]:
```suggestion
- Creating a new user
- Notifying a user when they get added to a shared album
- Informing other users about the addition of new assets to a shared album
```

File: docs/docs/administration/email-notification.md (hunk 1 above)
Comment (danieldietzler) [reply docs/docs/administration/email-notification.md:12]: I don't think we should remove this section completely, as this is a scenario many (if not most) users will run into. I agree with Alex in that we should move this section to a guide though

File: docs/docs/administration/email-notification.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+<TabItem value="Mobile" label="Mobile">
+
+Immich does not support this setting through the mobile app at this time.
+
+</TabItem>
+</Tabs>
+
+---
+
+:::danger plain password
```
Comment (danieldietzler) [inline docs/docs/administration/email-notification.md:63]:
```suggestion
:::danger Plain password
```

File: docs/docs/administration/email-notification.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+Email messages sent by a user contain the login password you set as plain text, it is recommended to choose a password that is not too easy but not too personal. In any case, unless otherwise specified, the user will be asked to enter a new password when logging in.
```
Comment (danieldietzler) [inline docs/docs/administration/email-notification.md:64]:
```suggestion
Invitation emails contain the initial password you set as plain text. It is recommended to choose a password that is neither too easy nor too personal. In any case, unless otherwise specified, the user will be asked to enter a new password when logging in.
```

File: docs/docs/administration/email-notification.md (hunk 1 above)
Comment (aviv926) [reply docs/docs/administration/email-notification.md:12]: So what sections would we like to leave on this page?

**Review by alextran1502 — APPROVED** [review]
